src/App.jsx

```diff
@@ -14,33 +16,128 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD_ALPHABET = 'alphabetically';
+const SORT_FIELD_LENGTH = 'length';
+const SORT_FIELD_ALPHABET_REVERSE = 'alphabetically_reverse';
+const SORT_FIELD_LENGTH_REVERSE = 'length_reverse';
+const SORT_FIELD_REVERSE = 'reverse';
```

Comment:
This approach has some potential problems.

1. Some sorting can be combined (it is OK to sort alphabetically, then by length, and then reverse), while some are incompatible (alphabetically and alphabetically_reverse)
2. Adding a new sort type (like color, price, etc) would NOT automatically allow reverse an order. You would need to add an extra sort order (color/color-reverse, etc)


What I suggest you to do here is:
```suggestion
//these are ok
const SORT_FIELD_ALPHABET = 'alphabetically';
const SORT_FIELD_LENGTH = 'length';
// add explicit field for the default sort - but keep it ''
const SORT_FIELD_UNSORTED = '';
```

reverseing can be handled by a separate boolean state `isReversed`